ee/api/max_tools.py

```diff
@@ -58,13 +58,13 @@ def create_and_query_insight(self, request: Request, *args, **kwargs):
         serializer = InsightsToolCallSerializer(data=request.data)
         serializer.is_valid(raise_exception=True)
         conversation = self.get_queryset().create(user=request.user, team=self.team, type=Conversation.Type.TOOL_CALL)
-        assistant = Assistant(
+        assistant = Assistant.create(
```

Comment:
Nitpick: Could be a stupid question, but why we don't type the response? I don't see any serializer for "action" either. Would adding something like `@extend_schema` add any value?

Reply:
It's just how you write endpoints in Django, `@action` with a `url_path=create_and_query_insights` creates a `/create_and_query_insights` endpoint.